components/test/brave_rewards/ui/reducers/rewards_reducer_test.ts

```diff
@@ -249,7 +252,8 @@ describe('rewards reducer', () => {
 
       const expectedState: Rewards.State = { ...defaultState }
       expectedState.inlineTip = {
-        twitter: false
+        twitter: false,
+        reddit: true
```

Comment:
Should this be `reddit: false`?

Reply:
No. reddit hasn't been set to false yet. Added additional tests to do so.